### PR TITLE
Example housekeeping + Support for EFM32GG_STK3700

### PR DESCRIPTION
**Pull request by AlessandroA**
@Patater @niklas-arm

### CODE REVIEW

File: source/main-hw.h [hunk 1]
```diff
@@ -72,7 +92,7 @@
         {EXTI,                sizeof(*EXTI),   UVISOR_TACLDEF_PERIPH}, \
         {GPIOG,               sizeof(*GPIOG),  UVISOR_TACLDEF_PERIPH}, \
         {SYSCFG,              sizeof(*SYSCFG), UVISOR_TACLDEF_PERIPH}, \
-        {(void *) 0x42000000, 0x01000000,      UVISOR_TACLDEF_PERIPH}, /* FIXME */ \
+        {(void *) 0x42000000, 0x01000000,      UVISOR_TACLDEF_PERIPH}, \
```
Comment (Patater) [inline source/main-hw.h]: This isn't about adding EFM32 support. Consider removing the FIXME in a separate commit.

File: README.md [hunk 2]
```diff
@@ -4,10 +4,11 @@ This is a simple example to show how to write a uVisor-secured threaded applicat
...
 
-| Target         | Toolchain | Public box LED | Secure box LED | User button   | Baud rate |
-|----------------|-----------|----------------|----------------|---------------|-----------|
-| `K64F`         | `GCC_ARM` | `LED_BLUE`     | `LED_RED`      | `SW2`         | 9600      |
-| `DISCO_F429ZI` | `GCC_ARM` | `LED1`         | `LED2`         | `USER_BUTTON` | 9600      |
+| Target            | Toolchain | Public box LED | Secure box LED | User button   | Baud rate |
+|-------------------|-----------|----------------|----------------|---------------|-----------|
+| `K64F`            | `GCC_ARM` | `LED_BLUE`     | `LED_RED`      | `SW2`         | 9600      |
+| `DISCO_F429ZI`    | `GCC_ARM` | `LED1`         | `LED2`         | `USER_BUTTON` | 9600      |
+| `EFM32GG_STK3700` | `GCC_ARM` | `LED1`         | `LED2`         | `SW0`         | 9600      |
```
Comment (Patater) [inline README.md]: In the commit message that adds EFM32GG, please mention that you fix https://github.com/ARMmbed/mbed-os-example-uvisor/issues/18 to auto-close it.

File: source/main-hw.h (hunk 1 above)
Comment (AlessandroA) [inline source/main-hw.h]: Done.

File: README.md (hunk 2 above)
Comment (AlessandroA) [inline README.md]: Done.

**Comment by AlessandroA** [comment]
PR updated @Patater

**Comment by AlessandroA** [comment]
This can be merged once our mbed OS updates are merged into mbed OS.

**Pull request merged**